Formula/corectl.rb

```diff
@@ -4,14 +4,8 @@ class Corectl < Formula
   head "https://github.com/TheNewNormal/corectl.git", :branch => "golang"
 
   stable do
```

Comment:
the `stable do ... end` block also needs to be removed, and the url and sha256 moved all the way up in the formula above the head url.